docs/nb/Simple_Scripts.ipynb

```diff
@@ -184,6 +184,36 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Meta Data :: specdb_meta\n",
+    "    specdb_meta -h\n",
+    "    usage: specdb_meta [-h] [--tol TOL] [-g GROUP] [--db_file DB_FILE] coord dbase\n",
+    "\n",
+    "    specdb_meta script v0.1\n",
+    "\n",
+    "    positional arguments:\n",
+    "      coord                 Coordinates, e.g. J081240.7+320809 or 122.223,-23.2322\n",
+    "                            or 07:45:00.47,34:17:31.1\n",
+    "      dbase                 Database [igmspec,uvqs,all,priv]\n",
+    "\n",
+    "    optional arguments:\n",
+    "      -h, --help            show this help message and exit\n",
+    "      --tol TOL             Maximum offset in arcsec [default=5.]\n",
+    "      -g GROUP, --group GROUP\n",
+    "                            Restrict on Group (e.g. BOSS_DR12)\n",
+    "      --db_file DB_FILE     Full path of db_file"
+   ]
```

Comment:
This does work:
`specdb_meta --db_file ../../igmspec/DB/IGMspec_DB_v02.hdf5 07:45:00.47,34:17:31.1 igmspec`